packages/components/psammead-headings/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.4.1   | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Vertical spacing should be defined outside of the component |
+| 0.4.0   | [PR#381](https://github.com/bbc/psammead/pull/381) Make headline 40px for 600px and above |
```

Comment:
Currently included as it'll be a conflict when https://github.com/bbc/psammead/pull/381 is merged.